src/main/scala/carts/ValidationError.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2022-2023 by Pervasive Cats S.r.l.s.
+ *
+ * All Rights Reserved.
+ */
+
+package io.github.pervasivecats
+package carts
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.NonNegative
+
+type Validated[A] = Either[ValidationError, A]
+
+type id = Long Refined NonNegative
```

Comment:
Why is this here? It needs its file. "Id" is a type, not a variable, so its name should start with an uppercase letter. A better name could be "IdLong"